src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -327,13 +340,31 @@ ActStatus ActionHandlers::process_ula(SM::ControlBlock& cb)
 	ue_ctxt->setSubscriptionStatus(ula_msg.subscription_status);
 	ue_ctxt->setNetAccessMode(ula_msg.net_access_mode);
 	ue_ctxt->setAccessRestrictionData(ula_msg.access_restriction_data);
-  ue_ctxt->setSubscribedApn(Apn_name(ula_msg.selected_apn));
-  log_msg(LOG_DEBUG, "Selected APN %s \n",ula_msg.selected_apn.val);
+	
+    ue_ctxt->setSubscribedApn(Apn_name(ula_msg.selected_apn));
+    log_msg(LOG_DEBUG, "Selected APN %s \n", ula_msg.selected_apn.val);
```

Comment:
Get rid of this log. If you have opportunity to do one more change. We really need to cleanup logs.

Reply:
Done.